package.json

```diff
@@ -248,34 +248,41 @@
     ]
   },
   "scripts": {
-    "vscode:prepublish": "npm run compile",
+    "vscode:prepublish": "npm run build",
+    "build:dev": "webpack --mode development",
+    "build": "webpack --mode production --devtool hidden-source-map",
     "compile": "tsc -p ./",
-    "watch": "tsc -watch -p ./",
-    "pretest": "npm run vscode:prepublish",
+    "watch": "webpack --mode development --watch",
+    "pretest": "npm run compile",
     "test": "node ./out/test/runTest.js",
-    "tslint": "gulp tslint",
+    "lint": "eslint . --ext .ts,.tsx",
+    "lint:fix": "eslint . --ext .ts,.tsx --fix",
```

Comment:
**Callout:** `npm run build` uses webpack is used to build (including assets) and transpiling Typescript to JS. Used before publishing. `npm run compile` only transpile Typescript to JS. Used for lauching extension in VSCode for testing or running unit tests